browser/widevine/widevine_utils.cc

```diff
@@ -9,8 +9,8 @@
 #include "brave/browser/widevine/widevine_permission_request.h"
 #include "brave/common/pref_names.h"
 #include "brave/grit/brave_generated_resources.h"
-#include "chrome/browser/permissions/permission_request_manager.h"
 #include "chrome/browser/profiles/profile.h"
+#include "components/permissions/permission_request_manager.h"
```

Comment:
deps?

Reply:
Already has //components/permissions dep in browser/widevine/BUILD.gn